lib/src/posts.dart

```diff
@@ -447,6 +450,16 @@ class _PostActionsViewState extends State<PostActionsView> with StateLocalizatio
       actions.add(IconButton(icon: Icon(Icons.keyboard_return), onPressed: _showOriginalPost, tooltip: l.showOriginalPost));
     }
 
+    if (widget.post.public) {
+      actions.add(
+        IconButton(
+            icon: Icon(
+                (Platform.isIOS ? Icons.ios_share : Icons.share)
+            ),
```

Comment:
Personally I got a big enough screen for


```suggestion
            icon: Icon(Platform.isIOS ? Icons.ios_share : Icons.share),
```
😛